extensions/ql-vscode/src/view/model-editor/MethodRow.tsx

```diff
@@ -88,38 +96,44 @@ const ModelableMethodRow = forwardRef<HTMLElement | undefined, MethodRowProps>(
   (props, ref) => {
     const {
       method,
-      modeledMethod,
+      modeledMethods: modeledMethodsArg,
```

Comment:
I think a name like this would be slightly better:

```suggestion
      modeledMethods: modeledMethodsProp,
```